### ISSUE TITLE
Support all input languages in the same run

### ISSUE THREAD

**Issue by schani** [quicktype/quicktype#344]
I'm not sure what the UI should be for this.  Maybe have options to specify input files from each language, such as:

```
quicktype --json input.json --schema input.schema --graphql query.graphql
```

And keep the `--src-lang` option and make it apply to all non-option arguments?

**Comment by schani** [comment]
Actually, we should probably just support GraphQL in the directory input feature and make that the suggested way of using quicktype.

**Comment by schani** [comment]
Fixed in 18e9b56e27ff219ba1f7d46e1571e70513d67d3b
